src/main/webapp/partials/instructorHelpArchiving.jsp

```diff
@@ -2,102 +2,113 @@
 <h4 class="text-color-primary" id="archiving">Archiving</h4>
 <div id="contentHolder">
   <br>
-  <ol style="list-style-type: none;">
-    <li id="archivingCourse">
-      <span class="text-bold">
-          <h3>1. Archiving a course</h3>
-      </span>
-      <div>
-        You can archive a course so that it doesn't appear in your home page anymore. You cannot edit, create feedback sessions or enroll students in an archived course.
-        <br><br>Go to your
-        <b>'Home'</b> page. You will see a separate panel for each course and a table of feedback sessions inside it. It should look something similar to this:
-        <br><br>
-        <div class="panel panel-primary">
-          <div class="panel-heading">
-            <div class="row">
-              <div class="col-sm-6">
-                <strong>
-                  [AI532] : Artificial Intelligence
-                </strong>
-              </div>
-              <div class="mobile-margin-top-10px col-sm-6">
-                <span class="mobile-no-pull pull-right">
-                  <div class="dropdown courses-table-dropdown">
-                    <button class="btn btn-primary btn-xs" type="button">
-                      Students
-                      <span class="caret"></span>
-                    </button>
-                  </div>
-                  <div class="dropdown courses-table-dropdown">
-                    <button class="btn btn-primary btn-xs" type="button">
-                      Instructors
-                      <span class="caret"></span>
-                    </button>
-                  </div>
-                  <div class="dropdown courses-table-dropdown">
-                    <button class="btn btn-primary btn-xs" type="button">
-                      Sessions
-                      <span class="caret"></span>
-                    </button>
+  <div class="bulletPointsIndent">
```

Comment:
`bulletPointsIndent` the class is not defined in `teammatesCommon.css` (although I don't know why). You should not use it or even delete all.

Reply:
I have deleted all.